client/tests/pw/e2e/workspaces_actions.spec.ts

```diff
@@ -0,0 +1,14 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { expect } from '@tests/pw/helpers/assertions';
+import { msTest } from '@tests/pw/helpers/fixtures';
+import { clearThenFillInputModal } from '@tests/pw/helpers/utils';
+
+msTest('Rename a workspace', async ({ connected }) => {
+  const firstWorkspace = connected.locator('.workspaces-container-grid').locator('.workspaces-grid-item').nth(1);
+  await firstWorkspace.locator('.card-option').click();
+  const popover = connected.locator('.popover-viewport');
+  await popover.getByRole('group').nth(0).click();
+  await clearThenFillInputModal(connected, 'New Workspace Name');
```

Comment:
A `fill()` directly doesn't work with the pre-selection? Do we really need to clear the input first before entering text?
Anyway, just add an argument with a default value to the existing function, no need to add an entire function for that.